pyproject.toml

```
@@ -6,8 +6,9 @@ requires = [
     "setuptools>=34.4",
 
     # Plugins
-    "setuptools_scm[toml]>=3.5",
-    "setuptools_scm_git_archive>=1.1",
+    "setuptools-scm >= 7.0.0,
```

This is a syntax error. Strings in TOML must have both opening and closing quotes.
```suggestion
    "setuptools-scm >= 7.0.0",
```